### PR TITLE
Cart and checkout PBC edits

### PR DESCRIPTION
**Pull request by AlexSlawinski**
A review of the cart and checkout PBC documentation.

## PR Description

TBD

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/cart-and-checkout/checkout-feature-overview/checkout-feature-overview.md [hunk 1]
```diff
@@ -22,18 +22,18 @@ The *Checkout* feature lets customers select single or multiple products and add
...
 
 With order thresholds, you can do the following:
 
-* Get buyers to place bigger orders, which can increase your sales.
-* Prevent waste of resources on small orders.
+* Ensure buyers place bigger orders, which can increase your sales.
+* Prevent the waste of resources on small orders.
 * Ensure that the cost of items sold is not too high for each transaction, which, in the long run, can make your business more profitable.
-* Support promotional campaigns—for example, by offering free shipping for orders above a threshold.
+* Support promotional campaigns — for example, by offering free shipping for orders above a threshold.
```
Comment (lenadoc) [inline docs/pbc/all/cart-and-checkout/checkout-feature-overview/checkout-feature-overview.md:36]: I believe in the original version it used to be an em dash, at least on the live documentation page it looks more like em dash :) Please check here https://developers.google.com/style/dashes and consider if the original dash should be left

File: docs/pbc/all/cart-and-checkout/cart-feature-overview/cart-notes-overview.md [hunk 2]
```diff
@@ -14,11 +14,11 @@ redirect_from:
   - /docs/scos/user/features/202204.0/cart-feature-overview/cart-notes-overview.html
 ---
 
-*Cart Notes* let Shop administrators include the **Notes** field on the **Cart** page. Customers can add notes to a particular item or the whole cart—for example, some special instructions about preparing and delivering an order.
+*Cart Notes* let Shop administrators include the **Notes** field on the **Cart** page. Customers can add notes to a particular item or the whole cart. for example, they can add special instructions about preparing and delivering an order.
```
Comment (lenadoc) [inline docs/pbc/all/cart-and-checkout/cart-feature-overview/cart-notes-overview.md:17]:
```suggestion
*Cart Notes* let Shop administrators include the **Notes** field on the **Cart** page. Customers can add notes to a particular item or the whole cart. For example, they can add special instructions about preparing and delivering an order.
```

File: docs/pbc/all/cart-and-checkout/extend-and-customize/cart-module-reference-information.md [hunk 3]
```diff
@@ -26,9 +26,9 @@ The `Cart` module in Zed has a cart operation class that handles cart operation
 
 * *Expands cart items*. Augments cart items with additional data (prices, product details, and product options).
 * *Persisting*. Storage provider stores items into the database and merges items if the same item is added. By default, it uses `NonPersistentProvider`.
-* *Item grouping*. `Cart` uses the item grouper to group items in the cart using a specially crafted group key, which is provided by the cart expander—for example, a list containing the same item coming from different merchants must be split into separate groups and shipped separately).
-* *Log operation message*. `Cart` writes messages to the messenger. These messages are returned to Yves—for example, validation messages, success, and failure messages.
-* *Cart recalculation*. It happens for each operation in cart recalculation. Cart amounts are reset and recalculated with newly added items.
+* *Item grouping*. `Cart` uses the item grouper to group items in the cart using a specially crafted group key, which is provided by the cart expander — for example, a list containing the same item coming from different merchants must be split into separate groups and shipped separately).
```
Comment (lenadoc) [inline docs/pbc/all/cart-and-checkout/extend-and-customize/cart-module-reference-information.md:29]: Same here for the em dash. Please check if the original em dash should be left

File: docs/pbc/all/cart-and-checkout/multiple-carts-feature-overview.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ redirect_from:
   - /docs/scos/dev/feature-walkthroughs/202204.0/multiple-carts-feature-walkthrough.html
 ---
 
-Sometimes customers need more than one shopping cart—for example, one for daily purchases and the other one for monthly expenses. The *Multiple Carts* feature lets you create and manage multiple shopping carts in one customer account.
+Sometimes customers need more than one shopping cart — for example, one for daily purchases and the other one for monthly expenses. The *Multiple Carts* feature lets you create and manage multiple shopping carts in one customer account.
```
Comment (lenadoc) [inline docs/pbc/all/cart-and-checkout/multiple-carts-feature-overview.md:19]: Same here for the em dash

File: docs/pbc/all/cart-and-checkout/persistent-cart-sharing-feature-overview.md [hunk 5]
```diff
@@ -16,11 +16,11 @@ redirect_from:
 
 The *Persistent Cart Sharing* feature lets company users generate the URL to share the cart with different access levels. To generate the link, you need to be logged in to a company account.
 
-URL is generated with the following structure: `your_domain/cart/preview/UIID`—for example, `http://mysprykershop.com/cart/preview/bea563fe-3f03-594e-8586-c5ae11e253fd`.
+The URL is generated with the following structure: `your_domain/cart/preview/UIID`—for example, `http://mysprykershop.com/cart/preview/bea563fe-3f03-594e-8586-c5ae11e253fd`.
 
-The cart can be shared with internal users—the users of the business unit, and external users—non-company users (friends, relatives, followers on social media). By accessing the provided URL, a user can perform the actions depending on the permissions level:
+The cart can be shared with internal users — the users of the business unit, and external users—non-company users (friends, relatives, followers on social media). By accessing the provided URL, a user can perform the actions depending on the permissions level:
```
Comment (lenadoc) [inline docs/pbc/all/cart-and-checkout/persistent-cart-sharing-feature-overview.md:21]: Same here for the em dash

**Review by lenadoc — CHANGES_REQUESTED** [review]

**Comment by AlexSlawinski** [comment]
hi @lenadoc, committed the changes - let me know if there is anything else I can do to improve the documentation!

**Review by lenadoc — APPROVED** [review]

**Pull request merged**